### PR TITLE
Features/first release limitations

### PR DESCRIPTION
**Pull request by c3d**
Some documentation about the first release limitations

### CODE REVIEW

**Comment by c3d** [comment]
@fitzthum Does this capture our discussion today?

@ariel-adam I added some terminology or rewording for the terms that I thought would be unknown to newcomers.

File: releases/release_notes_09_2022.md [hunk 1]
```diff
@@ -1,59 +1,84 @@
...
+## Limitations
+
+The following are known limitations of this release:
+
+- Platform support is currently limited, and rapidly changing
+  * AMD SEV is tested by the CI (with some limitations regarding attestation, see below)
+  * Intel TDX is expected to work, but not currently tested in the CI
+  * S390x is not supported by the COCO operator
+- Attestation and key brokering support is still under development
+  * THe disk-based key broker client (KBC) is still the primary method used for development, even if it will never be an acceptable approach in productino.
```
Comment (fitzthum) [inline releases/release_notes_09_2022.md:36]: `productino` -> `production`

**Review by fitzthum — APPROVED** [review]
One typo and this doesn't yet have the updates that we made in person, but once those are in I think it's good.

File: releases/release_notes_09_2022.md (hunk 1 above)
Comment (c3d) [reply releases/release_notes_09_2022.md]: Typo fixed, thanks. The changes we made in person should be there. What is missing in your opinion?

**Pull request merged**